bindings/jupyroot/CMakeLists.txt

```diff
@@ -10,7 +10,57 @@
 
 set(JupyROOTDirName python/JupyROOT)
 
-ROOT_LINKER_LIBRARY(JupyROOT src/IOHandler.cxx DEPENDENCIES Core CMAKENOEXPORT)
```

Comment:
why this doesn't appear anymore inside the loop as ```ROOT_LINKER_LIBRARY(${libname} src/IOHandler.cxx DEPENDENCIES Core CMAKENOEXPORT)```?

Reply:
That has been replaced by `add_library(${libname} SHARED src/IOHandler.cxx)` plus ` `target_link_libraries(${libname} ${python_library} Core ${CMAKE_DL_LIBS})`, I followed the example of how we build `libcppyy`, which also depends on python. Is there any advantage of using one or the other?

Reply:
By looking at the definition of ```ROOT_LINKER_LIBRARY``` I can't see anything important that we would miss, but maybe @oshadura knows better than me